public/apps/configuration/constants.tsx

```diff
@@ -106,17 +106,26 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/reports/instance/list',
   'cluster:admin/opendistro/reports/instance/get',
   'cluster:admin/opendistro/reports/menu/download',
+  'cluster:admin/opensearch/ml/create_model_meta',
   'cluster:admin/opensearch/ml/execute',
+  'cluster:admin/opensearch/ml/forward',
```

Comment:
Both forward/syncup seem like they apply to something rather than a top level permission, is this expected?

Reply:
Yes, good point, that part is not top level action. So we should only configure top level permission right? Will security plugin reject request if user don't have lower level action permission?